src/components/Card.js

```diff
@@ -1,10 +1,11 @@
 export default class Card {
-  constructor({ title, URL }, cardSelector, handleImageClick) {
-    console.log({ title, URL });
-    this._name = title;
-    this._link = URL;
+  constructor(data, cardSelector, handleImageClick, handleDeleteClick) {
+    this._data = data;
```

Comment:
* When I reload the page, all **black** hearts become **white** again.


you need to check `data.isLiked` to render the liked heart